### PR TITLE
should fix linux watch mode

### PR DESCRIPTION
**Pull request by sfranky**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
not fixed: it seems that with `watch -d ./qtop -FTG` I still get the bug

**Comment by fgeorgatos** [comment]
OK. one bug fixed, one bug received: 
* `./qtop.py -FGTw` truncates the top of the screen, ie. section 1 is scrolled up enough to hide it completely

**Comment by fgeorgatos** [comment]
also, `...and thank you for ..watching ;)` should not be visible up there when invoked with `watch -d`

File: qtop.py [hunk 1]
```diff
@@ -206,8 +208,15 @@ def calculate_term_size(config, FALLBACK_TERM_SIZE):
     Gets the dimensions of the terminal window where qtop will be displayed.
     """
     fallback_term_size = config.get('term_size', FALLBACK_TERM_SIZE)
+
     try:
-        term_height, term_columns = os.popen('stty size', 'r').read().split()
+        _command = subprocess.Popen('stty size', stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+        tty_size, _error = _command.communicate()
+        if not _error:
+            term_height, term_columns = [int(x) for x in tty_size.strip().split()]
```
Comment (fgeorgatos) [inline qtop.py]: `map(int, tty_size.strip().split())` ?

File: qtop.py [hunk 2]
```diff
@@ -2192,11 +2204,13 @@ class InvalidScheduler(Exception):
...
                     else:
                         output_partview_fp = display.show_part_view(file=output_fp, x=viewport.v_start, y=viewport.v_term_size)
+
                     cat_command = 'clear;cat %s' % output_partview_fp
                     _ = subprocess.call(cat_command, stdout=stdout, stderr=stdout, shell=True)
 
-                    read_char = wait_for_keypress_or_autorefresh(viewport, int(options.WATCH[0]) or KEYPRESS_TIMEOUT)
-                    control_qtop(viewport, read_char, cluster)
+                    read_char = wait_for_keypress_or_autorefresh(viewport, FALLBACK_TERMSIZE, int(options.WATCH[0]) or
+                                                                 KEYPRESS_TIMEOUT)
```
Comment (fgeorgatos) [inline qtop.py]: this is the way your editor tells you: "you have far too long function name" ;-)

File: qtop.py (hunk 1 above)
Comment (sfranky) [inline qtop.py]: afaik, list comps are preferred to map

**Comment by fgeorgatos** [comment]
lgtm, both in source and output; merging!

**Comment by fgeorgatos** [comment]
this resolves #248 & #206

**Pull request merged**